src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="formBig"
```

Comment:
for css common convention to name classes in kebab-case or little another convention (for example BEM) form-big__label form-big__label--active. But not camelCase.
formBig => form-big
please, check all such naming